server/routes/prediction.routes.js

```diff
@@ -554,7 +568,7 @@ module.exports = {
     let keys = Object.keys(req.body)
 
     // verify that only supported filter params are used
-    let validKeys = ['agency', 'office', 'numDocs', 'solNum', 'category_list', 'startDate', 'fromPeriod', 'endDate', 'toPeriod', 'noticeType']
+    let validKeys = ['agency', 'office', 'numDocs', 'solNum', 'category_list', 'startDate', 'fromPeriod', 'endDate', 'toPeriod', 'userEmail']
```

Comment:
Please replace the userEmail back with the notice type. We are not going to be filtering Solicitations based on user emails.

Reply:
The code completely removes any reference to userEmail filtering. Instead, it uses the notice type filter in applyBasicFilters